src/Donations/DonationsAdminPage.php

```diff
@@ -131,7 +132,16 @@ private function getForms()
             [
                 'value' => '0',
                 'text' => 'Any',
-            ]
+            ],
         ], $options);
     }
+
+    public function hasManualDonations()
+    {
+        if (Utils::isPluginActive('give-manual-donations/give-manual-donations.php')) {
+            return true;
+        }
+
+        return false;
```

Comment:
We do not need to return a boolean value explicitly. `Utils::isPluginActive` returns a boolean value.
```suggestion
       return Utils::isPluginActive('give-manual-donations/give-manual-donations.php')
      
```

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6638/commits/a5e19f64bc27610f090a20b63d0078a3ff462ad7